modules/gapi/samples/privacy_masking_camera.cpp

```diff
@@ -0,0 +1,216 @@
+#include <algorithm>
+#include <iostream>
+#include <cctype>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/infer.hpp>
+#include <opencv2/gapi/render.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/highgui.hpp>
+
+const std::string about =
+    "This is an OpenCV-based version of Privacy Masking Camera example";
+const std::string keys =
+    "{ h help |     | print this help message }"
+    "{ input  |     | Path to an input video file }"
+    "{ platm  |     | IE vehicle/license plate detection model (.xml) }"
+    "{ platd  | CPU | IE vehicle/license plate detection model device (for OpenVINO IE, e.g. CPU, GPU, VPU, ...) }"
+    "{ facem  |     | IE face detection model (.xml) }"
+    "{ faced  | CPU | IE face detection model device (for OpenVINO IE, e.g. CPU, GPU, VPU, ...) }"
+    "{ trad   |     | Run processing in traditional (non-pipelined) way }"
+    "{ noshow |     | Don't display UI (improves performance) }";
```

Comment:
Please use "false" default value for bool parameters.

Reply:
Done